README.md

```diff
@@ -8,12 +8,16 @@ This is done via using the [Lune API](https://docs.lune.co).
 
 ## How to install
 
-This utility is a NodeJS application and you should be able to run it on variety of OSes.
+This utility is a NodeJS application and you should be able to run it on variety of OSes
+(more details on this below).
 
 To install this utility and start using it, first perform the following steps:
 
 ### Installing from NPM (AKA "I just want to use the application")
 
+This is supported on Unix-like (Linux, BSD, macOS, Windows with WSL etc.) and Windows (without
+WSL).
```

Comment:
Why not just

`This is supported on Unix-like (Linux, BSD, macOS, etc.) and Windows.`

I feel like more is less here. This is for the less techy person.

Reply:
Sure